recipes/r-mpm/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = '1.0-22' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-mpm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: mpm_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/mpm_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/mpm/mpm_{{ version }}.tar.gz
+
+  sha256: d3ba4053cd57a189cb65c5fa20e6a4152374aead8c985254cb6e550e36e23272
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-kernsmooth
+    - r-mass
+
+  run:
+    - r-base
+    - r-kernsmooth
+    - r-mass
+
+test:
+  commands:
+    - $R -e "library('mpm')"  # [not win]
+    - "\"%R%\" -e \"library('mpm')\""  # [win]
+
+about:
+  home: http://mpm.r-forge.r-project.org
+  license: GPL (>= 2)
+  summary: 'Exploratory graphical analysis of multivariate data, specifically gene expression
+    data with different projection methods: principal component analysis, correspondence
+    analysis, spectral map analysis.'
+
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}\/lib\/R\/share\/licenses\/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]
+
+extra:
+  recipe-maintainers:
+    - MathiasHaudgaard
```

Comment:
can you add these guys as well? https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml